src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java

```diff
@@ -40,20 +42,20 @@ public CardService(CardRepository cardRepository, DeckRepository deckRepository,
             String email = user.getAccount().getEmail();
             List<Card> learningCards = new ArrayList<>();
             if (user.getAccount().getLearningRegime().equals(LearningRegime.BAD_NORMAL_GOOD_STATUS_DEPENDING)) {
-                learningCards = cardRepository.cardsForLearningWithOutStatus(email, deckId, CARDS_NUMBER);
-                if (learningCards.size() < CARDS_NUMBER) {
+                learningCards = cardRepository.cardsForLearningWithOutStatus(email, deckId, accountService.getCardsNumber());
+                if (learningCards.size() < accountService.getCardsNumber()) {
                     learningCards.addAll(cardRepository.cardsQueueForLearningWithStatus(email, deckId,
-                            CARDS_NUMBER - learningCards.size()));
+                            accountService.getCardsNumber() - learningCards.size()));
```

Comment:
Introduce variable equals
`accountService.getCardsNumber()`
and don't request the same value too often.

Reply:
Where to introduce that variable so that I could reach it from three different files?